src/main/java/nextstep/courses/service/SessionService.java

```diff
@@ -0,0 +1,37 @@
+package nextstep.courses.service;
+
+import nextstep.courses.domain.session.EnrollmentRepository;
+import nextstep.courses.domain.session.Session;
+import nextstep.courses.domain.session.SessionRepository;
+import nextstep.courses.domain.session.Sessions;
+import nextstep.payments.domain.Payment;
+import nextstep.payments.service.PaymentService;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class SessionService {
+    private final SessionRepository sessionRepository;
+    private final EnrollmentRepository enrollmentRepository;
+
+    public SessionService(SessionRepository sessionRepository, EnrollmentRepository enrollmentRepository) {
+        this.sessionRepository = sessionRepository;
+        this.enrollmentRepository = enrollmentRepository;
+    }
+
+    //TODO 추후 메서드의 파라미터는 사용하기에 따라 정리한다(현재 인수 4개)
+    public void enroll(Long courseId, int cohort, Long nsUserId, Payment payment) {
+        Sessions sessions = sessionRepository.findByCourseId(courseId);
+        Session session = sessions.findByCohort(cohort);
```

Comment:
```suggestion
    public void enroll(long sessionId, Long nsUserId, Payment payment) {
        Sessions session = sessionRepository.findById(sessionId);
```
특정 세션에 수강신청하는 것이라 굳이 전체 Session 목록을 가져올 필요는 없지 않을까?